main/lsp/request_dispatch.cc

```diff
@@ -1,5 +1,6 @@
 #include "common/Timer.h"
-#include "lsp.h"
```

Comment:
Clangd 9.0 has an auto-import feature, which is bad and leads to relative import paths like this one. I'm going to see if I can disable this "feature" for the Sorbet repo.